test/startservers.py

```diff
@@ -16,6 +16,20 @@
     default_config = 'test/boulder-config.json'
 processes = []
 
+def get_config(service):
+    """
+    returns the path of the configuration file. All configuration files need
+    to be of the form [service-name]-config.json. I.E. boulder-wfe-config.json
```

Comment:
As I said above, let's skip the `-config` suffix to save bytes. We can also tweak the rule a little bit to say it's the name of the binary, unless the name of the binary starts with `boulder-`, in which case we skip the prefix to save typing.